lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -220,6 +220,8 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
       element.appendChild(groupEl);
     }
 
+    // TODO: Handle passed w/ warnings in a unique clump like non-perf categories do
```

Comment:
This is ~~chicken nuggets~~ an issue

Reply:
ayup https://github.com/GoogleChrome/lighthouse/pull/9288